dimensioning.py

```diff
@@ -187,22 +187,55 @@ def getPoints(self):
                 
                     
     def calcab(self):
-        delta = self.p2 - self.p1
+        # get p1,p2 ordered for correct dimension direction
```

Comment:
Probably a neater way to do this but I have tested vertical,horizontal, parallel in all four quadrants